examples/meta/ruby/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ STRING(REGEX REPLACE "(.*)/narray.*$" "\\1" NARRAY_PATH ${NARRAY_LIB})
 
 LIST(APPEND EXCLUDED_RUBY_META_EXAMPLES
 			base_api-put_get
+      base_api-factory
```

Comment:
@vigsterkr same problem as for put: the check for SGMatrix/object bails

Reply:
(so I blacklisted it for now)

Reply:
but that should be fixed in develop, so rebase over develop, otherwise i dont understand what's the problem

Reply:
fixed in develop? I don't remember us fixing that SWIG bug? Did I miss it?
https://github.com/shogun-toolbox/shogun/issues/4177